recipes/solr-deploy.rb

```diff
@@ -267,10 +267,15 @@
     action [:stop]
 end
 bash "Copy latest index from EFS" do
+    user account_name
     code <<-EOS
         rsync -a --delete #{efs_data_dir}/ #{real_data_dir}/
-        LATEST_INDEX=`ls -dtr #{efs_data_dir}/data/#{core_name}/data/snapshot.* |tail -1`
-        rsync $LATEST_INDEX/ #{real_data_dir}/data/#{core_name}/data/index/
+        CORE_DATA="#{real_data_dir}/data/#{core_name}/data"
```

Comment:
how many snapshots do we keep on the efs?
could we move from the full file on efs but use a s3 pointer file to reduce efs costs also?

Reply:
The sync script, on export, removes all snapshots except the current one (`solr-sync.sh` line 90).

We can probably just drop EFS and use S3 without too much trouble. I didn't do it here because it wasn't needed, but it should be fairly straightforward. We don't use EFS for anything that demands high I/O performance; it's just putting timestamps in heartbeat files, and passing snapshots in the background.